news/_posts/releases/2020-09-01-prestashop-1-7-7-0-beta2-release.md

```diff
@@ -0,0 +1,203 @@
+- [Fix session tables charset](https://github.com/PrestaShop/PrestaShop/pull/20122)
+- [Error when installing without debug mode disabled](https://github.com/PrestaShop/PrestaShop/issues/19754)
+
+**Circuit breaker:**
+The [Circuit Breaker](https://martinfowler.com/bliki/CircuitBreaker.html) design pattern provides strategies and functions to manage the unavailability of the third-party services, read more about it [here](https://build.prestashop.com/news/resilient-php-applications/)
+- [CB - Page Login - must_have.xml API call in AdminController](https://github.com/PrestaShop/PrestaShop/issues/12831)
+- [CB - Page Login - List modules call API for Recommended Modules](https://github.com/PrestaShop/PrestaShop/issues/12833)
+- [CB - Page Login - call native.xml in Admin controller](https://github.com/PrestaShop/PrestaShop/issues/12825)
+- [CB - Page Login - call native_all.xml in AdminController](https://github.com/PrestaShop/PrestaShop/issues/12826) 
+
+## Deprecations
+- [Deprecate hookcount and add missing hook in hook.xml](https://github.com/PrestaShop/PrestaShop/pull/19657)
+- [Order hooks deprecated](https://github.com/PrestaShop/PrestaShop/pull/20311)
+- [Product::supplier_reference deprecated](https://github.com/PrestaShop/PrestaShop/pull/20198)
+- [Addons related calls have no more effect](https://github.com/PrestaShop/PrestaShop/pull/19461)
+
+## BC breaks
+- [Increase precision for DECIMAL fields in database](https://github.com/PrestaShop/PrestaShop/pull/18532)
+- [Calculator::calculateFees argument ignored](https://github.com/PrestaShop/PrestaShop/pull/20482)
+- [ModuleTabRegister constructor has a new parameter](https://github.com/PrestaShop/PrestaShop/pull/19903)
+- [Old order page files removed](https://github.com/PrestaShop/PrestaShop/pull/18984)
+
+## Changelog
+
+130+ pull requests have been merged since the first beta release and 100+ issues have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. 
+
+If you are looking for more details about all changes and new features expected in 1.7.7, the [1.7.7 beta version release note](https://build.prestashop.com/news/prestashop-1-7-7-0-beta-release/) is the perfect article for you to read!
+
+
+## Download
+
+You can download PrestaShop 1.7.7.0 beta 2 here:  
+
+[Download PrestaShop 1.7.7.0 beta 2 now!](https://www.prestashop.com/en/developers-versions){: .btn .btn-lg .btn-success}
+{: .text-center}
+
+
+## How to help finish the 1.7.7
+
+Your feedback on this beta 2 is essential, the regressions you find right now are as many bugs less on the final version and less patches versions. It’s also fewer problems on your (or your customer’s) online store. 
+
+Here’s how:
+
+- Install it and test that there is no regression compared to the 1.7.6 version
+- If you develop modules or themes, test them on this version
+- Test the migrated back-office order pages: the add order page and the order detail page
+- Test the auto-upgrade plugin
```

Comment:
Please explain briefly how, providing manually the beta 2 zip to the plugin.

Reply:
We have the documentation explaining how to install PrestaShop: http://doc.prestashop.com/display/PS17/Installing+PrestaShop, I can link it

Reply:
ah sorry, I commented the wrong line.